content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -36,84 +49,248 @@ This section of the vehicle form relates to the person who is currently driving
+    associated company.
+  - :guilabel:`Delivery Address`: select this option to add a preferred address for all deliveries.
+    When added to the form, this address is selected by default when delivering an order to the
+    associated company.
+  - :guilabel:`Follow-up Address`: select this option to add a preferred address for all follow-up
+    correspondence. When added to the form, this address is selected by default when sending
+    reminders about overdue invoices.
+  - :guilabel:`Other Address`: select this option to add any other necessary addresses for the
+    company.
+
+.. image:: new_vehicle/create-contact.png
+   :align: center
+   :alt: The create contact form with all parts filled in.
+
+Depending on the :guilabel:`Contact Type`, some optional fields may not be visible. The available
+fields are identical to the fields in the :ref:`general information
+<fleet/new_vehicle/general-info>` section of the new driver form.
+
+Add any notes to the :guilabel:`Internal notes...` section of the form.
+
+After entering all of the information, click either :guilabel:`Save & Close` to add the one new
+contact, or :guilabel:`Save & New` to add the current address record and create another address
+record.
+
+As contacts are added to this tab, each contact appears in a separate box, with an icon indicating
+what type of contact is listed.
+
+.. example::
+   An :guilabel:`Invoice Address` displays a dollar sign icon inside that specific address box,
+   whereas a :guilabel:`Delivery Address` displays a truck icon inside.
+
+   .. image:: new_vehicle/contacts-address.png
+     :align: center
+     :alt: The create contact form with all parts filled in.
+
+Sales & purchase tab
+********************
+
+Enter the following sales and purchasing information for the various sections below. Depending on
+the other installed applications, additional fields and sections may appear. The following are all
+default fields for the *Fleet* application only.
+
+Sales
+^^^^^
+
+- :guilabel:`Salesperson`: using the drop-down menu, select the user who is the main point of
+  contact for sales with this driver's company.
+
+  This person **must** be an internal user of the company, meaning they can log into the database as
+  a user.
+
+Misc
+^^^^
+
+- :guilabel:`Company ID`: if the company has an ID number **other than** its *tax ID*, enter it in
+  this field.
+- :guilabel:`Reference`: enter any text to give more information regarding the contact person. This
+  is an internal note to provide any additional information.
+
+  .. example::
+     A company has several people with the same name, John Smith. The :guilabel:`Reference` field
+     could state `John Smith at X205 - purchaser` to provide additional details.
+
+Internal notes tab
+******************
+
+Add any notes that pertain to the driver or any other necessary information, in this tab.
 
 Vehicle section
```

Comment:
You appear to jump away from the 'Create Driver' form breakdown and *back* to the 'new form' breakdown here -- however, in the runbots I'm using for my review, there are 2 more tabs in t he 'Create Driver' pop-up that haven't been focused on up to this point: Partner Assignment and Membership

Can you please double-check and add explanations about these tabs, as well, if needed? Thanks!

Reply:
Those are also from other apps (I do not know off the top of my head which ones make that appear, but they do not appear for a fleet-only DB).